tests/testthat/test-DFFilterStates.R

```diff
@@ -251,21 +253,27 @@ testthat::test_that(
       varlabels = character(0),
       keys = character(0)
     )
-    testthat::expect_identical(dffs$ui_add_filter_state("id", data.frame()), div("no sample variables available"))
-    testthat::expect_identical(dffs$ui_add_filter_state("id", data.frame(A = numeric(0))), div("no samples available"))
+    testthat::expect_identical(
+      dffs$ui_add_filter_state("id", data.frame()),
+      div("no sample variables available")
+    )
+    testthat::expect_identical(
+      dffs$ui_add_filter_state("id", data.frame(A = numeric(0))),
+      div("no samples available")
+    )
   }
 )
 
 # Format
 testthat::test_that("$format() is a method of DFFilterStates", {
-  testthat::expect_error(isolate(
+  testthat::expect_no_error(isolate(
```

Comment:
it's not especially important but I would try to isolate as little as possible (so we get failed tests if we add reactivity to somewhere new - but it's not worth changing this at this time

DF <- DFFilterStates$new(...)
testthat::expect_no_error(isolate(DF$format

Reply:
`isolate` IS necessary here because there is a `reactiveVal` in `private$state_list` (formerly `queue`) now:
```
DFFilterStates$new(
      input_dataname = "test",
      output_dataname = "test",
      datalabel = character(0),
      varlabels = "test",
      keys = "test"
    )$format()
```
```
Error in `.getReactiveEnvironment()$currentContext()`:
! Operation not allowed without an active reactive context.
• You tried to do something that can only be done from inside a reactive consumer.
Run `rlang::last_error()` to see where the error occurred.
```